libcaf_net/caf/net/web_socket/framing.hpp

```diff
@@ -172,6 +174,9 @@ class CAF_NET_EXPORT framing : public octet_stream::upper_layer,
   /// Assembles fragmented payloads.
   binary_buffer payload_buf_;
 
+  /// Validated UTF-8 range for fragmented text payloads
+  size_t validation_point = 0ul;
```

Comment:
I'm not quite sure whether I understand what this doing from the comment and declaration. Is this just the offset where we've stopped last time? In that case, naming it offset might be clearer:

```suggestion
  /// Stores where to resume the UTF-8 input validation.
  size_t validation_offset_ = 0;
```